docs/sources/next/javascript-api/k6-experimental/browser/frame/ishidden.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 'isHidden(selector[, options])'
+excerpt: 'Browser module: frame.isHidden(selector[, options]) method'
+---
+
+# isHidden(selector[, options])
+
+{{% admonition type="warning" %}}
+
+Use locator-based [`locator.isHidden([options])`](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/locator/ishidden/) instead.
+
+{{% /admonition %}}
+
+Checks if the element is `hidden`.
+
+<TableWithNestedRows>
+
+| Parameter      | Type    | Default | Description                                                                                                                                                |
+| -------------- | ------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| selector       | string  | `''`    | A selector to search for an element. If there are multiple elements satisfying the selector, the first will be used.                                       |
+| options        | object  | `null`  |                                                                                                                                                            |
+| options.strict | boolean | `false` | When `true`, the call requires selector to resolve to a single element. If given selector resolves to more than one element, the call throws an exception. |
+
+</TableWithNestedRows>
+
+### Returns
+
+| Type | Description                                      |
+| ---- | ------------------------------------------------ |
+| bool | `true` if the element is `hidden`, else `false`. |
```

Comment:
Small suggestion, feel free to skip it if you think the current form is OK:
```suggestion
| bool | `true` if the element is `hidden`, `false` otherwise. |
```